assets/components/stripeInlineForm/stripeInlineForm.jsx

```diff
@@ -0,0 +1,108 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import { StripeProvider, Elements, CardElement, injectStripe } from 'react-stripe-elements';
+import { getStripeKey } from 'helpers/paymentIntegrations/stripeCheckout';
+import { type Currency } from 'helpers/internationalisation/currency';
+import { type Status } from 'helpers/switch';
+import SvgArrowRightStraight from 'components/svgs/arrowRightStraight';
+import Switchable from 'components/switchable/switchable';
+import PaymentError from 'components/switchable/errorComponents/paymentError';
+
+
+// ----- Types -----//
+
+/* eslint-disable react/no-unused-prop-types */
+type PropTypes = {
+  stripeIsLoaded: void => void,
+  isStripeLoaded: boolean,
+  currency: Currency,
+  isTestUser: boolean,
+  callback: () => mixed,
+  switchStatus: Status,
+};
+/* eslint-enable react/no-unused-prop-types */
+
+
+// ---- Auxiliary functions ----- //
+
+const setupStripeInlineForm = (stripeIsLoaded: void => void) => {
+  const htmlElement = document.getElementById('#stripe-js');
```

Comment:
Does this work? Shouldn't it be `document.getElementById('stripe-js')`? Perhaps it doesn't make a difference 🤷‍♀️